great_expectations/expectations/set_based_column_map_expectation.py

```diff
@@ -41,7 +42,24 @@
 logger = logging.getLogger(__name__)
 
 
+@public_api
 class SetColumnMapMetricProvider(ColumnMapMetricProvider):
+    """Base class for all SetColumnMapMetrics.
+
+    SetColumnMapMetric classes inheriting from SetColumnMapMetricProvider are ephemeral,
+    defined by their `set` attribute, and registered during the execution of their associated SetColumnMapExpectation.
+
+    In some cases, subclasses of MetricProvider, such as SetColumnMapMetricProvider, will already
+    have correct values that may simply be inherited by Metric classes.
+
+    Attributes:
```

Comment:
For some reason the rendering doesn't like `Attributes:`
```suggestion
    Args:
```